packages/react-examples/src/react-charting/HeatMapChart/HeatMapChart.PiecewiseScale.Example.tsx

```diff
@@ -0,0 +1,172 @@
+import * as React from 'react';
+import { HeatMapChart, IHeatMapChartProps } from '@fluentui/react-charting';
+
+interface IHeatMapChartPiecewiseScaleExampleState {
+  width: number;
+  height: number;
+}
+
+export class HeatMapChartPiecewiseScaleExample extends React.Component<{}, IHeatMapChartPiecewiseScaleExampleState> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      width: 450,
+      height: 350,
+    };
+  }
+  public render(): React.ReactNode {
```

Comment:
Lets add this example to docsite as well if it hasn't been already added.

Reply:
Shouldn't I remove it? The basic example already shows how piecewise scale can be used. There are 4 (more than 2) domain values and the same number of range values.

Reply:
If it is already covered then you can remove it.

Reply:
removed it